app/views/courses/_course_actions.html.haml

```diff
@@ -0,0 +1,6 @@
+- if logged_in?
+  - if @course.date_and_time.future?
+    = link_to "RSVP", course_rsvp_path(@course), class: 'button'
+- else
+  = link_to "Sign up", new_member_path, class: 'button'
+  = link_to "Log in", login_path, class: 'button'
```

Comment:
I abstracted out the Login / RSVP functionality on every WECM show page just like was already done on the workshop show page.